README.md

```diff
@@ -465,6 +469,31 @@ If the daemon has been installed, live stats of CPU/system load monitoring and o
 
 `auto-cpufreq --stats`
 
+## Battery charging thresholds
+
+As of v2.2.0, battery charging thresholds can be set in the config file. This enforces your battery to start and stop charging at defined values
```

Comment:
Let's make turn [v2.0.0](https://github.com/AdnanHodzic/auto-cpufreq/releases) into a link.

Reply:
Would also be good to include  [this](https://github.com/AdnanHodzic/auto-cpufreq/pull/645#issuecomment-1942159310). Maybe as a last line for this section to say something like: _"Please note: only --monitor and --stats have battery threshold function"_

Or maybe it could be omitted, my only concern is that folks will ask it as a question. Alternatively it could be added as a comment for config file as part of battery threshold?

Reply:
Since I got a moment now to do the release, I'll just merge the changes and make this update myself.

Reply:
Done. The link should work after the release is made, since it links directly to tag v2.2.0